scripts/exregional_run_prepstart.sh

```diff
@@ -661,15 +661,50 @@ if [ ${SFC_CYC} -eq 1 ] || [ ${SFC_CYC} -eq 2 ] ; then  # cycle surface fields
         fi
       done
     fi
+    surface_file_path=$bkpath
+
+# check if there are surface file in continue cycle data space:
+    if [ "${restart_suffix_find}" == "missing" ] || [ "${restart_prefix_find}" == "missing" ]; then
+      surface_file_path=${CONT_CYCLE_DATA_ROOT}/surface
```

Comment:
Is this "CONT_CYCLE_DATA_ROOT" defined somewhere for non-wcoss2 machines?

Reply:
This is for wcoss2 only now.